zerver/lib/event_queue.py

```diff
@@ -453,6 +493,7 @@ def setup_event_queue():
     send_restart_events(immediate=settings.DEVELOPMENT)
 
 def fetch_events(query):
+    # type: (Dict[str, Any]) -> Dict[str, Any]
```

Comment:
@sharmaeklavya2 ah, yeah i changed it to `Mapping` since nothing was changing the `query`